src/m365/planner/commands/task/task-details-get.ts

```diff
@@ -0,0 +1,56 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import commands from '../../commands';
+import { Task } from '../../Task';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: string;
+}
+
+class PlannerTaskDetailsGetCommand extends GraphItemsListCommand<Task> {
+  public get name(): string {
+    return commands.TASK_DETAILS_GET;
+  }
+
+  public get description(): string {
+    return 'Retrieve the details of the specified planner task';
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    const requestOptions: any = {
+      url: `${this.resource}/v1.0/planner/tasks/${encodeURIComponent(args.options.id)}/details`,
+      headers: {
+        accept: 'application/json;odata.metadata=none'
+      },
+      responseType: 'json'
+    };
+
+    request
+      .get(requestOptions)
+      .then((res: any): void => {
+        logger.log(res);
+        cb();
+      }, (err: any): void => this.handleRejectedODataJsonPromise(err, logger, cb));
+  }
+
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-i, --id <id>'
```

Comment:
Since we renamed the command and the last noun is now `details`, this option should be named `taskId`